pwnlib/util/misc.py

```diff
@@ -359,3 +370,8 @@ def register_sizes(regs, in_sizes):
             smaller[r] = [r_ for r_ in l if sizes[r_] < sizes[r]]
 
     return lists.concat(regs), sizes, bigger, smaller
+
+
+if __name__ == "__main__":
```

Comment:
This block is unnecessary, please remove it.  See `TESTING.md` for how to run tests.